examples/fhir/lab_order.json

```diff
@@ -0,0 +1,118 @@
+{
+  "resourceType": "Bundle",
+  "id": "278637ea-386c-4e24-8aad-d3b2ec088332",
+  "type": "message",
+  "timestamp": "2023-02-23T15:42:22.877-06:00",
+  "entry": [
+    {
+      "resource": {
+        "resourceType": "MessageHeader",
+        "id": "76d92393-4290-49e5-88f2-d4cdfe2e0b6f",
+        "eventCoding": {
+          "system": "http://terminology.hl7.org/CodeSystem/v2-0003",
+          "code": "O21",
+          "display": "OML - Laboratory order"
+        }
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Patient",
+        "id": "infant-twin-1",
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race",
+            "extension": [
+              {
+                "url": "text",
+                "valueString": "Asian"
+              }
+            ]
+          }
+        ],
+        "identifier": [
+          {
+            "type": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                  "code": "MR",
+                  "display": "Medical Record Number"
+                }
+              ]
+            },
+            "value": "MRN7465737865"
+          }
+        ],
+        "name": [
+          {
+            "use": "official",
+            "family": "Solo",
+            "given": [
+              "Jaina"
+            ]
+          }
+        ],
+        "gender": "female",
+        "birthDate": "2017-05-15",
+        "_birthDate": {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/patient-birthTime",
+              "valueDateTime": "2017-05-15T11:11:00-05:00"
+            }
+          ]
+        },
+        "multipleBirthInteger": 1,
+        "contact": [
+          {
+            "relationship": [
+              {
+                "coding": [
+                  {
+                    "system": "http://terminology.hl7.org/CodeSystem/v2-0131",
+                    "code": "N",
+                    "display": "Next of kin"
+                  }
+                ]
+              }
+            ],
+            "name": {
+              "family": "Organa",
+              "given": [
+                "Leia"
+              ]
+            },
+            "telecom": [
+              {
+                "system": "phone",
+                "value": "+31201234567"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "resource": {
```

Comment:
We may also need to include the `category`.  There is a SNOMED code for `laboratory procedure`.

Consider adding a `requester`

Reply:
Both the `category` and `requester` have a cardinality of optional.  Is there something thats making you feel we need to add these?  I basically completely cooked this up from the required fields (if you couldn't tell).

Reply:
Just what I've gleaned from other examples I've found.  But yes, those are not required, and they don't appear to map to an OML, but I think they add context.

Reply:
Added the `category`.